apps/web/src/app/events/[slug]/page.tsx

```diff
@@ -51,39 +55,21 @@ export default async function Page({ params }: { params: { slug: string } }) {
                 <Text variant={"headS"} bold>
                   Agenda
                 </Text>
-                <LectureCard
-                  from={"2023-08-16T17:19:08.398Z"}
-                  to={"2023-08-16T17:19:08.398Z"}
-                  title={"AAA"}
-                  description={"aaa"}
-                  speakers={[
-                    {
-                      name: "Jan Kowalski",
-                      description: "Januszex CEO",
-                    },
-                    {
-                      name: "Karol Kowalski",
-                      description: "Woźny",
-                    },
-                  ]}
-                />
-                <div className={styles.separator} />
-                <LectureCard
-                  from={"2023-08-16T17:19:08.398Z"}
-                  to={"2023-08-16T17:19:08.398Z"}
-                  title={"BBB"}
-                  description={"bbb"}
-                  speakers={[
-                    {
-                      name: "Jan Kowalski",
-                      description: "Januszex CEO",
-                    },
-                    {
-                      name: "Karol Kowalski",
-                      description: "Woźny",
-                    },
-                  ]}
-                />
+
+                {lectures.length > 0 &&
+                  lectures.map((lecture, index) => (
+                    <>
+                      {index !== 0 && <div className={styles.separator} />}
```

Comment:
W takich przypadkach key też jest wymagany
```suggestion
                      {index !== 0 && <div className={styles.separator} key={lecture.id + "-separator"} />}
```